src/doc/en/developer/coding_basics.rst

```diff
@@ -353,7 +353,9 @@ information. You can use the existing functions of Sage as templates.
 
        - ``n`` -- integer
 
-       - ``p`` -- prime integer (default: `2`); coprime with ``n``
+       - ``p`` -- prime integer (default: `2`); coprime with `n`
+
+       - ``check`` -- (default: ``True``) specifies whether to check primality
```

Comment:
```suggestion
       - ``check`` -- boolean (default: ``True``); specifies whether to check primality
```

Reply:
I also prefer tight lists, but there is no consensus on that it seems.

Reply:
I prefer spaced lists. Right... no consensus.

Line 358 is by purpose. Type of an input variable may be omitted. In that case, semicolon seems redundant.

Reply:
Hm, I see. It would be nice to have an example for booleans too, because people may write it as "bool" or "Boolean". Maybe we should add an extra example with default value ``None`` that has no type.

Reply:
Something like
```
-  ``algorithm`` -- (default: ``None``) the name of the algorithm to use
```
However I think it is problematic to start a sentence with a parenthetical remark.

Reply:
How about
```
- ``algorithm`` -- default: ``None``; the name of the algorithm to use
```
?

Reply:
> Something like
> 
> ```
> - ``algorithm`` -- (default: ``None``) the name of the algorithm to use
> ```

Yes. This example may be better.

> However I think it is problematic to start a sentence with a parenthetical remark.

I regard that as a convention (for default value). We should put the parenthetical phrase somewhere in the line. I see no better place. I think that this choice is already established.

Reply:
Ok, let's leave it at that.

Reply:
Thanks. Done.